components/motor/ulnstepper/28byj-48.go

```diff
@@ -0,0 +1,372 @@
+// Package unipolarfivewirestepper implements a GPIO based
+// stepper motor (model: 28byj-48) with uln2003 controler.
+package unipolarfivewirestepper
```

Comment:
let's keep the name the same as the folder -- ulnstepper

Reply:
you don't like this name?